<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_ejercicio1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejercicio 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ejercicio2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejercicio 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ejercicio3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejercicio 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ejercicio4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejercicio 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Salir">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>1001</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Sistema de Gestión de Tareas</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>681</width>
        <height>491</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>50</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingrese Tipo de tarea</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_tipodetareas">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>50</y>
          <width>131</width>
          <height>26</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Tareas Diarias</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tareas Semanales</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingrese Descripciones de la tarea</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_descripciones">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>90</y>
          <width>291</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Crear Tarea</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Titulo de Tarea</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_titulodelatarea">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>161</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>210</y>
          <width>391</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingrese Fecha</string>
        </property>
       </widget>
       <widget class="QPushButton" name="GuardarTarea">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>410</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>20</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ver Tareas</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_mostratareas">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>641</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_vertareas">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>390</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Ver Tareas</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear Tarea</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Ver tarea</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Sistema de Figuras Geométricas</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_figura">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Circulo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Triangulo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cuadrado</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_cambiarfigura">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>460</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Cambiar Figura</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>421</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Editor de Texto con Formato</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_ejercicio3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>821</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_negrita">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>NEGRITA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cursiva">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>CURSIVA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_subrayado">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>90</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>SUBRAYADO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Sistema de Gestión de Inventario</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>961</width>
        <height>411</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_7">
       <widget class="QLineEdit" name="nombreproducto">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>40</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre del producto</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="precioproducto">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>80</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Precio del Producto</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="descripcion">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>130</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Descripcion</string>
        </property>
       </widget>
       <widget class="QComboBox" name="tipoproducto">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>230</y>
          <width>151</width>
          <height>26</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Electronico</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ropa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alimento</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>230</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Tipo de Producto</string>
        </property>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget_4">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>60</y>
          <width>341</width>
          <height>201</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="page_9">
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>20</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Electronico</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Modelo</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="modelo">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>50</y>
            <width>113</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>100</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Garantia</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="garantia">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>100</y>
            <width>113</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_10">
         <widget class="QLabel" name="label_18">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>20</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Ropa</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_19">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>60</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Marca</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>50</y>
            <width>113</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_11">
         <widget class="QLabel" name="label_20">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>20</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Alimento</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_21">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>50</y>
            <width>151</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Fecha de Produccion</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>50</y>
            <width>131</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_22">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>151</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Fecha de Vencimiento</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>90</y>
            <width>131</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="Guardarproducto">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>310</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_8">
       <widget class="QPushButton" name="mostrarproducto">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>360</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Ver</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="mo">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>100</y>
          <width>421</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>30</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Mostrar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_12">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>90</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>90</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingrese nombre del producto</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>200</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>20</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>50</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
